.circleci/workflow-queue-v1.sh

```diff
@@ -100,30 +111,48 @@ if [[ ! "${current_pipeline_start_time}" ]]; then
   exit 1
 fi
 
+fetch_older_pipelines "${current_pipeline_start_time}"
+pipeline_workflow_ids=$__
+printf "%s\n%s\n\n" "Currently running workflow_id are:" "${pipeline_workflow_ids}"
```

Comment:
Run `fetch_older_pipelines` once before `while` loop to find all running workflows. Same set of workflow IDs are iterated thru inside the `while` loop.